recipes/default.rb

```diff
@@ -103,7 +103,8 @@ class Chef::Resource # rubocop:disable all
     end
   else
     # We need to download the rpm
-    rpm_package = "https://www.rabbitmq.com/releases/rabbitmq-server/v#{node['rabbitmq']['version']}/rabbitmq-server-#{node['rabbitmq']['version']}-1.noarch.rpm"
+    #rpm_package = "https://www.rabbitmq.com/releases/rabbitmq-server/v#{node['rabbitmq']['version']}/rabbitmq-server-#{node['rabbitmq']['version']}-1.noarch.rpm"
+    rpm_package = "#{node['rabbitmq']['mirror']}/rabbitmq-server-#{node['rabbitmq']['version']}-1.noarch.rpm"
```

Comment:
It's a great idea to have the ability to put in other mirrors, but to @stevecstian 's point, this is another PR.